fluent/asyncsender.py

```diff
@@ -132,5 +135,13 @@ def _send_loop(self):
         finally:
             self._close()
 
+    def _call_queue_overflow_handler(self, discarded_bytes):
+        try:
+            if self._queue_overflow_handler:
```

Comment:
I don't like checks like that in the critical paths, as it's going to be repeated every call.
Can we push it up into the constructor like we do here https://github.com/fluent/fluent-logger-python/blob/master/fluent/handler.py#L86?